### PR TITLE
Add zh_TW locale

### PR DESCRIPTION
**Pull request by dlackty**

### CODE REVIEW

**Comment by KrauseFx** [comment]
That's great, thanks :+1:

File: config/locales/zh_TW.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# To use a different locale, set it with `I18n.locale`:
+#
+#     I18n.locale = :es
+#
+# This would use the information in config/locales/es.yml.
+#
+# To learn more, please read the Rails Internationalization guide
+# available at http://guides.rubyonrails.org/i18n.html.
+
+en:
```
Comment (BalestraPatrick) [inline config/locales/zh_TW.yml]: Shouldn't this be zh_TW instead of en? Just tried it and now the language is always Chinese.

Comment (0xced) [inline config/locales/zh_TW.yml]: Yes, [absolutely](http://twitter.com/KrauseFx/status/660137248776523776).

Comment (KrauseFx) [inline config/locales/zh_TW.yml]: @0xced Tweets should be displayed inline in GitHub :+1:

Comment (KrauseFx) [inline config/locales/zh_TW.yml]: Commit message: "Fixes http://twitter.com/KrauseFx/status/660137248776523776"

**Pull request merged**